homeassistant/components/tellduslive/config_flow.py

```diff
@@ -69,17 +77,15 @@ def __init__(self):
                 }
             return self.async_create_entry(
                 title=host, data={
+                    KEY_HOST: host,
+                    KEY_SCAN_INTERVAL: self._scan_interval.seconds,
                     KEY_SESSION: session,
-                    KEY_HOST: host
                 })
 
-        def get_auth_url(session):
-            return session.authorize_url
-
```

Comment:
Sorry @MartinHjelmare but couldn't find any way to mock this function from the tests if I didn't extract it to a separate function.